File: src/python/Concat2.py
import sys

def main():
    f = open(sys.argv[1])
    lines = f.readlines()
    f.close()
    if len(sys.argv) > 2 and sys.argv[2] == '--dry-run':
        return
    concatenated = ''.join(lines)

if __name__ == '__main__':
    main()

File: src/perl/Concat2.pl
use strict;
use warnings;

sub main {
    open FH, $ARGV[0];
    my @lines = <FH>;
    close FH;
    if ($#ARGV > 0 and $ARGV[1]) {
        return;
    }
    my $concatenated = join '', @lines;
}

main();

